sites/friday/src/get-started/here/set-resources.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Managing resources
+sidebarTitle: Resources
+weight: 40
+description: Flexible resources on {{% vendor/name %}} are easy to use. Find all you need to know about resources allocation there.
+---
+
+To define allocated resources to your services and apps, you need to define how much CPU, memory, and disk to assign to the various containers.
+Back in your terminal, run:
+
+```bash {location="Terminal"}
+{{% vendor/cli %}} resources:set
+```
+
+This will launch an interactive prompt to walk you through setting up your application's resources:
+```bash {location="Terminal"}
+{{% vendor/cli %}} resources:set
+Resource configuration for the project app (123456azerty), environment main (type: production):
++-----------------------+---------+---------+-------------+-----------+-----------+
```

Comment:
this terminal output isnt correct. Since the resources are automatically assigned a default at first push, the size, CPU, and memory values will reflect the default values.